### PR TITLE
[MIG] barcodes_generator_abstract: Migration to 14.0

### PR DESCRIPTION
**Pull request by Zar21**
Superseeds https://github.com/OCA/stock-logistics-barcode/pull/313

### CODE REVIEW

**Review by hparfr — APPROVED** [review]
LGTM

**Comment by hugho-ad** [comment]
LGTM :+1:

**Review by hugho-ad — APPROVED** [review]

File: barcodes_generator_abstract/i18n/es_ES.po [hunk 1]
```diff
@@ -0,0 +1,198 @@
+# Translation of Odoo Server.
```
Comment (pedrobaeza) [inline barcodes_generator_abstract/i18n/es_ES.po]: Remove this file

File: barcodes_generator_abstract/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  class barcode_rule(models.Model):
+      _inherit = 'barcode.rule'
+
+      generate_model = fields.Selection(selection_add=[('my.model', 'My Model')])
+
+Eventually, you should inherit your model view adding buttons and fields.
+
+Note
+----
```
Comment (pedrobaeza) [inline barcodes_generator_abstract/readme/USAGE.rst]:
```suggestion
~~~~
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by francesco-ooops** [comment]
@legalsylvain could you find a way to get the readme file regenerated? image urls are broken (while they're not in "configuration" file)

thanks :)

**Comment by legalsylvain** [comment]
Image path is bad.you should set "../static/"

**Pull request merged**